activestate.ci.yaml

```diff
@@ -0,0 +1,49 @@
+constants:
+  - name: UPX_TARGET
+    if: eq .OS.Name "Windows"
+    value: ./.github/deps/Windows/bin/upx.exe
+  - name: UPX_TARGET
+    if: eq .OS.Name "MacOS"
+    value: ./.github/deps/macOS/bin/upx
+  - name: UPX_TARGET
+    if: eq .OS.Name "Linux"
+    value: ./.github/deps/Linux/bin/upx
+scripts:
+  - name: ci-build
+    language: bash
+    description: Builds the project with the host OS as the target OS.
+    value: |
+      set -e
+      $constants.SET_ENV
+      export CGO_ENABLED=0
+
+      go build -tags "$GO_BUILD_TAGS" -o $BUILD_TARGET_DIR/$constants.BUILD_TARGET $constants.CLI_BUILDFLAGS $constants.CLI_PKGS
+      $UPX_TARGET $BUILD_TARGET_DIR/$constants.BUILD_TARGET
```

Comment:
Rather than introducing what amounts to duplicates of the existing scripts I think we can reduce the maintainability headache by modifying the existing scripts with something along the lines of:

```
if [ $CI != "" ]; then
  $UPX_TARGET $BUILD_TARGET_DIR/$constants.BUILD_TARGET
fi
```